recipes/aws-c-io/all/conanfile.py

```diff
@@ -39,9 +39,12 @@ def configure(self):
         del self.settings.compiler.libcxx
 
     def requirements(self):
-        self.requires("aws-c-cal/0.5.11")
-        if self.settings.os in ["Linux", "FreeBSD"]:
-            self.requires("s2n/1.0.11")
+        self.requires("aws-c-cal/0.5.12")
+        if self.settings.os in ["Linux", "FreeBSD", "Android"]:
+            if tools.Version(self.version) <=  "0.10.6":
+                self.requires("s2n/1.0.11")
```

Comment:
I think it is enough to use a newer s2n 1.1.0 for all versions. Tried 0.10.5 with 1.1.0, it compiles and nothing breaking in release notes.

Reply:
This is a point I have been struggling with.
I thought it would be best to reduce the number of elements that affect the existing version as much as possible.
My idea seems to be conservative, so I follow yours.